npm/nameSpaceController.go

```diff
@@ -385,6 +410,13 @@ func (nsc *nameSpaceController) syncUpdateNameSpace(newNsObj *corev1.Namespace)
 			metrics.SendErrorLogAndMetric(util.NSID, "[UpdateNamespace] Error: failed to delete namespace %s from ipset list %s with err: %v", newNsName, labelKey, err)
 			return err
 		}
+		// {IMPORTANT} The order of compared list will be key and then key+val. NPM should only append after both key
+		// key + val ipsets are worked on. 0th index will be key and 1st index will be value of the label
+		// (TODO) need to remove this ordering dependency
```

Comment:
Can we resolve this issue in this PR? As I understood this issue and we discussed this before, but if possible, it is better to resolve this issue to avoid future possible issue.

Reply:
current GetLabelKVFromSet returns key and value. So better to returns (string, string) in the GetLabelKVFromSet 
 function and use "removedKey" and "removedValue" in this function